src/components/form/dt-multi-select/dt-multi-select.js

```diff
@@ -367,6 +416,14 @@ export class DtMultiSelect extends HasOptionsList(DtFormBase) {
             ?required=${this.required}
           />
         </div>
+        ${this.allowAdd
+          ? html`<button
+          class="input-addon btn-add"
+          @click=${this._addRecord}
+          >
+            <dt-icon icon="mdi:account-plus-outline"></dt-icon>
```

Comment:
This icon isn't actually loading for me, and it looks like it's because the dt-icon component is not included. You'll have to add it in the imports at the top: `import '../../icons/dt-icon.js';`